doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -237,25 +237,13 @@ nRF Desktop
   * The DTS description for board targets with a different DTS overlay file for each build type to isolate the common configuration that is now defined in the :file:`app_common.dtsi` file.
     The :ref:`zephyr:nrf54h20dk_nrf54h20` board configuration has been updated.
 
-  * The :ref:`nrf_desktop_ble_conn_params` with the following changes:
-
-    * Fixed the Bluetooth LE connection parameters update loop (NCSDK-30261) that replicated if an nRF Desktop dongle without Low Latency Packet Mode (LLPM) support was connected to an nRF Desktop peripheral with LLPM support.
-    * The module now waits until a triggered Bluetooth LE connection parameters update is completed before triggering subsequent updates for a given connection.
-    * Improved the log to also display the information if USB is suspended.
-      The information is needed to determine the requested connection parameters.
-    * The module now uses non-zero Bluetooth LE peripheral latency while USB is suspended.
-      This is done to prevent peripheral latency increase requests from :ref:`nrf_desktop_ble_latency` on peripheral's end.
-    * The module reverts the USB suspended Bluetooth LE connection parameter update when USB cable is disconnected.
-
   * The :ref:`nrf_desktop_ble_scan` to always use a connection interval of 10 ms for peripherals without Low Latency Packet Mode (LLPM) support if a dongle supports LLPM and more than one Bluetooth LE connection.
     This is required to avoid Bluetooth Link Layer scheduling conflicts that could lead to HID report rate drop.
 
-nRF SoC flash driver
---------------------
+  * The :ref:`nrf_desktop_failsafe` to use the Zephyr :ref:`zephyr:hwinfo_api` driver for getting and clearing the reset reason information (see the :c:func:`hwinfo_get_reset_cause` and :c:func:`hwinfo_clear_reset_cause` functions).
```

Comment:
Discussed with @b-gent. The release is based on NCS 2.9.99 (release from main after NCS 2.9). We can inform that: `This release adds the following nRF54H20 functionalities on top of the nRFH20 features included in the 2.9.0 release <link>" ` and only describe features introduced after NCS 2.9.0 (that approach may be simpler and more intuitive).

If we opt for doing it, most of nRF Desktop's release notes can be removed (see https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.9.0.html#nrf_desktop for reference) Seems that only these 2 notes at the bottom should remain here.

Reply:
After further discussions, we will not be doing it this way.
For the sake of good UX (not jumping between two release notes documents) and because we already did this in rc-1 docs - we want to have all nRF54H20 items mentioned here that were part of the 2.9.0 release plus everything that went into the release branch after 2.9.0. This is the original approach that was planned and we decided to stick to it.
FYI @kapi-no